scripts/qa_crowbarsetup.sh

```diff
@@ -4642,6 +4641,9 @@ function onadmin_testsetup
     # use local cache whenever possible
     oncontroller mount_localreposdir
 
+    # Check that the deployment did not cause failcounts
+    oncontroller_check_crm_failcounts disallowskipfailcount
```

Comment:
Need a space between oncontroller and the rest

Reply:
ah, right.. lets see if this works..